src/pages/Mypage/MainContent/ApplyStatus/ApplyStatus.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './ApplyStatus.scss';
+
+const ApplyStatus = () => {
+  return (
+    <div className="ApplyStatus">
+      <h1 className="applyStatusTitle">지원 현황</h1>
```

Comment:
```suggestion
      <h1 className="title">지원 현황</h1>
```
PR에 질문주신 부분과 동일한 부분인 것 같은데 이 컴포넌트 안에서는 굳이 applyStatus라고 명시할 필요는 없을 것 같네요!
applyStatus 컴포넌트 안의 ApplyStatus className안에 위치하고 있다면 title이라고만 기입해도 될 것 같습니다! 어차피 ApplyStatus에 대한 내용인걸 모두가 아니까요!